cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/WorkflowHttpHandlerTest.java

```diff
@@ -460,6 +460,7 @@ public void testMultipleWorkflowInstances() throws Exception {
   }
 
   private void verifyMultipleConcurrentRuns(Id.Program workflowId) throws Exception {
+    verifyProgramRuns(workflowId, ProgramRunStatus.RUNNING.name(), 1);
```

Comment:
verifyProgramRuns actually wait for 2 runs if you check the implementation. However I agree contract of this method is not clear. Will fix in separate PR.